includes/handler/class-accept.php

```diff
@@ -55,11 +55,11 @@ public static function handle_accept( $accept, $user_id ) {
 		 * Fires after an ActivityPub Accept activity has been handled.
 		 *
 		 * @param array              $accept  The ActivityPub activity data.
-		 * @param int                $user_id The local user ID.
+		 * @param array              $user_id The local user IDs.
```

Comment:
```suggestion
		 * @param int[]              $user_ids The local user IDs.
```
Let's be ruthless on all these do_action updates